tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import os
+from collections.abc import Iterator
+from unittest import mock
+
+import pytest
+
+
+@pytest.fixture()
+def _mock_env() -> Iterator[None]:
```

Comment:
consider yielding the `os.environ`, so that the return value doesn't have to be `None`